src/FileFormats/MOF/write.jl

```diff
@@ -174,22 +177,28 @@ function moi_to_object(
     set = MOI.get(model, MOI.ConstraintSet(), index)
     dual_start = MOI.get(model, MOI.ConstraintDualStart(), index)
     primal_start = MOI.get(model, MOI.ConstraintPrimalStart(), index)
-    object = OrderedObject()
+    keys = Symbol[]
+    values = Any[]
     if F != MOI.VariableIndex
         name = MOI.get(model, MOI.ConstraintName(), index)
         if name != ""
-            object["name"] = name
+            push!(keys, :name)
+            push!(values, name)
         end
     end
-    object["function"] = moi_to_object(func, name_map)
-    object["set"] = moi_to_object(set, name_map)
+    push!(keys, :function)
+    push!(values, moi_to_object(func, name_map))
+    push!(keys, :set)
+    push!(values, moi_to_object(set, name_map))
     if !isnothing(dual_start)
-        object["dual_start"] = dual_start
+        push!(keys, :dual_start)
+        push!(values, dual_start)
     end
     if !isnothing(primal_start)
-        object["primal_start"] = primal_start
+        push!(keys, :primal_start)
+        push!(values, primal_start)
     end
-    return object
+    return NamedTuple{Tuple(keys)}(values)
```

Comment:
Instead of two separate vectors, you can use:
```julia
julia> pairs = Pair{Symbol,Any}[]
Pair{Symbol, Any}[]

julia> push!(pairs, :name => "abc")
1-element Vector{Pair{Symbol, Any}}:
 :name => "abc"

julia> push!(pairs, :dual_start => 0.0)
2-element Vector{Pair{Symbol, Any}}:
       :name => "abc"
 :dual_start => 0.0

julia> NamedTuple(pairs)
(name = "abc", dual_start = 0.0)
```